data/remnant/remnant outfits.txt

```diff
@@ -40,6 +40,7 @@ outfit "Inhibitor Cannon"
 		"hit force" 39
 		"slowing damage" .5
 	description `Nearly a century ago, a Remnant scout exploring space north of the Ember Waste came under fire from an alien fleet armed with weapons that interfered with the scout's engines. The story says that the Remnant captain calmly told her crew, "We aren't going home until we steal that weapon." That alien technology became the basis of the Inhibitor Cannon.`
+	description `	While the Remnant were intrigued by the metaphysics that enabled the slowing effect, they decided that they needed to focus on developing a longer ranged weapon to compliment the close-quarters Thrasher. As such, while the inhibitor has but a mere shadow of its inspiration's warping effects, it is a very effective weapon.`
```

Comment:
At this point you're downplaying the slowing effect, which, again, I think is a lot stronger than people are giving it credit for, even if it doesn't immobilize the target.
```suggestion
	description `	Although not powerful enough to fully immobilize a ship, the Inhibitor is still a formidable weapon. The Inhibitor is excellent at controlling engagement distances, keeping targets at range by both pushing them backward with the impact of each shot while also slowing their approach by interfering with their engines.`
```